<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family:Century gothic;
	font-size:10px;
	font-weight: bold; 
}
QPushButton{
	border-radius:15px;
	background:blue;
	color:white;
}

QPushButton:hover{
	border-radius:15px;
	background:#49ebff;
	color:#333;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
*{
	font-family:Century gothic;
	font-size:24px;
}
QFrame{
border-radius:10px;
}
QPushButton{
	border-radius:50px;
	background:blue;
}
QToolButton{
	border-radius:50px;
	background:blue;

}

QLabel{
	color:white;
}

QLineEdit{
	background:transparent;
	border:none;
	color:#717072;
	border-bottom:1px solid #717072;
}
#groupBox{
background:url(:/images/bg.jpg);
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>180</y>
       <width>671</width>
       <height>661</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#D3D3D3;
border-radius:25px;</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>240</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>S'IDENTIFIER</string>
      </property>
     </widget>
     <widget class="QPushButton" name="login_button">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>580</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	border-radius:25px;
	background:blue;
	color:white;
}

QPushButton:hover{
	border-radius:25px;
	background:#49ebff;
	color:#333;
}

</string>
      </property>
      <property name="text">
       <string>Se Connecter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="username_input">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>350</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>R520</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_input">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>460</y>
        <width>321</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::EchoMode::Password</enum>
      </property>
      <property name="placeholderText">
       <string>Mot de Passe</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
